boards/arm/mimxrt1050_evk/mimxrt1050_evk.dts

```diff
@@ -33,7 +33,11 @@
 #elif defined(CONFIG_CODE_HYPERFLASH)
 		zephyr,flash = &hyperflash0;
 #endif
+#if defined(CONFIG_DEVICE_CONFIGURATION_DATA)
```

Comment:
This logic seems backwards to me. I think the DCD should be enabled as a result of someone choosing to link data into SDRAM.

Reply:
@MaureenHelm Does DCD work for QSPI version of the board?

Reply:
It should